html/static/templates/home.html.tpl

```diff
@@ -21,12 +24,16 @@
                                     <p>{{ .Text }}</p>
                                 </div>
                             </div>
-                            <a class="btn btn-outline-secondary" href="{{ .Link }}"><i class="fa fa-book fa-fw" aria-hidden="true"></i> Read</a>
+                            <a class="btn btn-outline-secondary mt-auto ms-auto" href="{{ .Link }}"><i class="fa fa-book fa-fw" aria-hidden="true"></i> {{ gettext "Read" }}</a>
                         </div>
                     </div>
                 </div>
             {{ end }}
 
     </div>
 
+    <div class="container position-absolute bottom-0 start-50 translate-middle-x py-2 d-flex justify-content-between bd-highlight mb-3" style="margin-bottom: 0 !important;">
+        <small class="text-muted">{{ .Version }} ({{ .BuildTime }})</small>
```

Comment:
If built without proper `ldflags` this displays an empty `()` at the bottom. Could you check for them to be available and hide those parentheses if not available?

Reply:
Oops, indeed I had forgotten to put some checks.
It's now corrected (see 04c6c1a). :)